be/src/www/bootstrap/js/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
If new thirdparty file is added, "LICENSE.txt" will be updated

Reply:
Added.